README.md

```diff
@@ -14,3 +14,5 @@ Our brains communicate through neurotransmitters and their activity emits electr
 Current devices in the medical industry are usually not accessible by doctors due to their high pricing. They are also complicated to use. The idea of the device is to integrate it into a headband and focus on signals that can be obtained through the frontal lobe.
 
 A difference to existing projects like OpenBCI is that it will not be necessary to 3D print large headsets. Instead we are focusing on creating a device that collects as much data as possible through the forehead. To achieve this goal we are using high-grade sensors and flexible electronics.
+
+The Android Application need to support from API level 16. Some parts of the desktop version of the application might not support it, so be sure to use alternatives for the same and implement them effectively.
```

Comment:
No need of this. There is no relation between desktop app features with API16. Two different platforms.

Reply:
No, I know that but for example I was talking about using ConcurrentLinkedDeque whose library could be imported and used in the Desktop version without any issue but in the case of Android the same is not possible for API level 16, we need to annotate it with "Required API level 21". Sir, I hope you get my point.

Reply:
But still @CloudyPadmal sir as you told I have removed this change from this PR.